pkg/client/cli/command.go

```diff
@@ -96,34 +107,41 @@ func Command(ctx context.Context) *cobra.Command {
 	if !IsServerRunning() {
 		myName = "Telepresence (daemon unavailable)"
 	}
-
 	rootCmd := &cobra.Command{
 		Use:  "telepresence",
 		Args: OnlySubcommands,
 
-		Short:         myName,
-		Long:          help,
-		RunE:          RunSubcommands,
-		SilenceErrors: true, // main() will handle it after .ExecuteContext() returns
-		SilenceUsage:  true, // our FlagErrorFunc will handle it
+		Short:              myName,
+		Long:               help,
+		RunE:               RunSubcommands,
+		PostRunE:           checkLegacyCmd,
```

Comment:
I added it as a PostRun so that this logic didn't get gross (in case we want to remove this at some point in the future).  But that being said, when users use telepresence 1 commands, the help text will be printed to the terminal (from RunSubCommands) and then my command will run. 

Example output:
```
...
  other Telepresence flags:
      --no-report   turn off anonymous crash reports and log submission on failure

Use "telepresence [command] --help" for more information about a command.

You used a telepresence 1 command that roughly translates to the following:
telepresence intercept echo-easy --port 9090 -- sleep 1
running...
Launching Telepresence Daemon v2.2.2-14-g56008474-1620355231 (api v3)
Connecting to traffic manager...
...
```
I considered this to not be a negative since I think while we want tp1 commands to work, we do want people to eventually migrate to using tp2 commands so the help text will maybe push them to explore.  But if you think it's messy, I can probably merge it into the RunSubcommands function.